doc/Requirements Specification/Parts/Discussion.lyx

```diff
@@ -248,7 +248,7 @@ gls{PCM}
 
 \end_inset
 
- describe the software correctly, completely and exactly like implemented
+ describe the software correctly, completely, and exactly like implemented
```

Comment:
Are you sure, that there has to be a comma?
In german, no comma would be needed.